docs/reference/index-modules/allocation/filtering.asciidoc

```diff
@@ -59,8 +59,21 @@ PUT test/_settings
 ------------------------
 // TEST[s/^/PUT test\n/]
 
-If you specify multiple filters, all conditions must be satisfied for shards to
-be relocated. For example, to move the `test` index to `big` nodes in `rack1`,
+If you specify multiple filters, all `require` type conditions and no `exclude` type conditions must be satisfied for shards to
```

Comment:
Thanks @DaveCTurner makes sense about removing the second example. Maybe just replace this section by:

```
If you specify multiple filters the following conditions must be satisfied simultaneously by a node in order for shards to be relocated to it:

* If any `require` type conditions are specified, all of them must be satisfied
* If any `exclude` type conditions are specified, none of them may be satisfied
* If any `include` type conditions are specified, at least one of them must be satisfied

```

then, make the example below use `require` as I did here and revert the addition I made to `include` to keep it short and sweet?

Reply:
Sounds good to me (mod wrapping to 80 columns ofc)

Reply:
Thanks, adjusted accordingly now :)